espnet2/bin/s2t_inference.py

```diff
@@ -575,6 +575,12 @@ def decode_long(
         text_prev = init_text
         while offset < len(speech):
             logging.info(f"Current start time in seconds: {offset / fs:.2f}")
+            if offset + segment_len > len(speech) and len(segment) / fs < 0.2:
```

Comment:
Should this be placed after `segment = speech[offset : offset + segment_len]`? Otherwise, `segment` is the previous one.
Also, we can make 0.2 as an argument in this function.
I think we do not need `offset + segment_len > len(speech)`. If `segment` is shorter than the segment length, this condition must be true.

Reply:
This is a mistake during copy-paste. Thanks!